cns-hook-util

```diff
@@ -236,20 +237,21 @@ function clean_challenge {
         mdata_delete "triton.cns.acme-challenge"
     else
         local vmuuid
-        case "$(getservice "${domain}")" in
-        cloudapi|adminui|docker|cmon|grafana)
-            local alias
-            alias="$(getservice "${domain}")0"
-            vmuuid="$(vmadm lookup alias="$alias")"
-            verifyvm "$domain" "$vmuuid"
-            echo "{\"remove_customer_metadata\":[\"triton.cns.acme-challenge\"]}" | \
-                vmadm update "$vmuuid"
-            ;;
-        *)
-            # We should never hit this becuase it would have been filtered out
-            # much earlier.
-            echo "ERROR: Unknown serivce: $vmservice"
-            ;;
+        vmservice="$(getservice "${domain}")"
+        case "$vmservice" in
+            cloudapi|adminui|docker|cmon|grafana|manta)
+                local alias
+                alias="$(getservice "${domain}")0"
+                vmuuid="$(vmadm lookup alias="$alias")"
+                verifyvm "$domain" "$vmuuid"
+                echo "{\"remove_customer_metadata\":[\"triton.cns.acme-challenge\"]}" | \
+                    vmadm update "$vmuuid"
+                ;;
+            *)
+                # We should never hit this becuase it would have been filtered out
```

Comment:
`s/becuase/because/`

(If you want to update, while here, not a blocker though)